include/recording.h

```diff
@@ -85,11 +85,21 @@ struct recording_method {
 };
 
 extern const struct recording_method *selected_recording_method;
+extern const struct recording_method methods[2];
```

Comment:
May I suggest a different approach here... instead of using strcmp() all over the place to make the case distinctions, which will inevitably lead to messier and messier code over time, either
1) retain `selected_recording_method` to point to a single recording method, and create a new 3rd recording method "all" (or "both") in the array `methods` that uses wrapper functions to call the functions for both `pcap` and `proc` methods, or
2) eliminate `selected_recording_method` and replace it with something like a bit field, e.g. 0x1 for proc, 0x2 for pcap, 0x3 for both, and then use bit tests in `_rm` to determine which functions to call. The `kernel_support` flag would then have to be carried separately.